src/pages/Register/index.tsx

```diff
@@ -1,118 +1,144 @@
 import React, { useState } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
 import { Link } from 'react-router-dom'
-import { Form, Col } from 'react-bootstrap'
+import { Form, Col, Row } from 'react-bootstrap'
 
 import HalfCircle from '../../components/HalfCircle'
 import CustomSvgIcon from '../../components/CustomSvgIcon'
 import starsLady from '../../media/star-lady.svg'
 import CustomButton from '../../components/CustomButton'
 import Loader from '../../components/Loader'
 import Message from '../../components/Message'
-import { registerEmployerRequest } from '../../redux/actions'
-import { AppState } from '../../types'
+import {
+  registerEmployerRequest,
+  registerJobSeekerRequest,
+} from '../../redux/actions/'
+import { AppState } from '../../redux/types'
+import FormContainer from '../../components/FormContainer'
+//import "./Register.scss";
 
 const Register = () => {
-  const [validated, setValidated] = useState(false)
+  const [isJobSeeker, setIsJobSeeker] = useState(false)
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
   const [confirmPassword, setConfirmPassword] = useState('')
   const [message, setMessage] = useState('')
 
-  // const employerRegister = useSelector((state: AppState) => state.employerRegister)
-  // const { loading, error, credentials } = employerRegister
+  const employer = useSelector((state: AppState) => state.employer)
+  const { loading, error } = employer
 
   const dispatch = useDispatch()
 
+  const handleChange = () => {
+    if (!isJobSeeker) {
+      setIsJobSeeker(true)
+      console.log(isJobSeeker)
+    } else {
+      setIsJobSeeker(false)
+      console.log(isJobSeeker)
+    }
+  }
```

Comment:
Why do we need this ? Shouldn't there be a button where user can choose to register as a `jobSeeker` or `employer` ? Then that button will set the state of `role`. Then `onSubmit`, based on the state of `role`, we can dispatch the relevant action to the reducer: `employer` or `jobSeeker`

Reply:
This works though, but maybe both of them want their own buttons. :))